### PR TITLE
Adding support for comments in Aptfiles.

### PR DESCRIPTION
**Pull request by dmerrick**
This simple fix allows you to include comments in your `Aptfile`.

It also helps make the README more readable.

### CODE REVIEW

**Comment by earnold** [comment]
This would be super helpful!

File: README.md [hunk 1]
```diff
@@ -20,8 +20,11 @@ heroku buildpacks:add --index 1 https://github.com/heroku/heroku-buildpack-apt
 
 #### Aptfile
 
+    # you can list packages
```
Comment (renanlecaro) [inline README.md]: This really needs to be in the readme, i was wondering what all those 3 lines where all about, i thought it was just one package being described in some weird way

**Comment by edmorley** [comment]
I've added a test. The Travis failure is due to #16 and is also present on `master`.

**Comment by dmerrick** [comment]
Thanks, @edmorley !

**Pull request merged**